packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -124,7 +125,7 @@ export async function activate(context: vscode.ExtensionContext) {
   const commands = registerCommands(context);
   context.subscriptions.push(commands);
 
-  context.subscriptions.push(await registerTestView(testOutlineProvider));
+  context.subscriptions.push(await registerTestView());
```

Comment:
No longer passing local instance of ApexTestOutlineProvider, now using testOutlineProvider that's exported from './views/testOutlineProvider';